examples/typed-component/sample_application_config.yaml

```diff
@@ -8,10 +8,7 @@ spec:
   workloadRefs:
     - apiVersion: core.oam.dev/v1alpha2
       kind: ContainerizedWorkload
-      name: example-component
-    - apiVersion: core.oam.dev/v1alpha2
-      kind: ContainerizedWorkload
-      name: example-component2
```

Comment:
why remove

Reply:
Because Application does not contain a component named example-component2 or example-component1。It contains a component named web-service-component.

Reply:
I see, thanks